frontend/src/features/documents/list/DocumentListView.test.tsx

```diff
@@ -70,8 +73,8 @@ describe('Document List View', () => {
       isLoading: false,
       parentId: 0,
       relationshipType: DocumentRelationshipType.FILES,
-
       documentResults: mockDocumentsResponse(),
+      deleteHandle: deleteMock,
```

Comment:
GH doesn't let me comment on non-modified line but this test 'should have the Documents type in the component' and the next one 'should NOT have the Documents status in the component' seem to be doing the same? kinda confused by the test duplication...

Reply:
good catch, these tests were written for the list view and do not seem to provide value. I will remove one.